downloads/swan-lake-release-notes/swan-lake-2201.8.3/RELEASE_NOTE.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: ballerina-left-nav-release-notes
+title: Swan Lake Update 8 (2201.8.3) 
+permalink: /downloads/swan-lake-release-notes/2201.8.3/
+active: 2201.8.3
+---
+
+## Overview of Ballerina Swan Lake Update 8 (2201.8.3)
+
+<em>Swan Lake Update 8 (2201.8.3) is the third patch release of Ballerina 2201.8.0 (Swan Lake Update 8) and it includes a new set of bug fixes to the runtime and developer tooling.</em>
+
+## Update Ballerina
+
+Run the command below to update your current Ballerina installation directly to 2201.8.3 by using the [Ballerina Update Tool](/learn/update-tool/) as follows.
+
+```
+$ bal dist pull 2201.8.3
+```
+
+## Install Ballerina
+
+If you have not installed Ballerina, download the [installers](/downloads/#swanlake) to install.
+
+## Runtime updates
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.8.3 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.8.3+label%3AType%2FBug+is%3Aclosed+label%3ATeam%2FjBallerina).
+
```

Comment:
```suggestion

## Library updates
### Breaking changes

#### OpenAPI Tool
- Transitioned code generation from representing OpenAPI Specification byte format of string type as Ballerina `byte[]` to using Ballerina `string` type.

**OpenAPI Sample**
```Yaml
components:
  schemas:
       ByteExample:
	      type: string
              format: byte
```
**Old Generated Ballerina code**
```ballerina
type ByteExample byte[];
```
**New Generated Ballerina code**
```ballerina
type ByteExample string;
```
- Changed the client resource/remote code generation of `http:Response` to `nil` return type when the operation of OpenAPI specification has a response with no content.

**OpenAPI Sample**
```yaml
 /users/drafts/{id}:
    delete:
      operationId: deleteDraft
      parameters:
        - in: path
          name: id
          required: true
          schema:
            type: string
      responses:
        "200":
          description: Successful response //No content type
```
**Old Generated client resource/remote function**
```ballerina
    resource isolated function delete users/drafts/[string id]() returns http:Response|error {
        string resourcePath = string `/users/drafts/${getEncodedUri(id)}`;
        http:Response response = check self.clientEp-> delete(resourcePath);
        return response;
    }
```
**New Generated client resource/remote function**
```ballerina
resource isolated function delete users/drafts/[string id]() returns error? {
        string resourcePath = string `/users/drafts/${getEncodedUri(id)}`;
        return self.clientEp-> delete(resourcePath);
}
```
### Bug fixes

To view bug fixes, see the GitHub milestone for Swan Lake Update 2201.8.3 of the repositories below.
- [OpenAPI](https://github.com/ballerina-platform/ballerina-library/issues?q=is%3Aissue+label%3Amodule%2Fopenapi-tools+label%3AType%2FBug+milestone%3A2201.8.3+is%3Aclosed)
```